src/types.ts

```diff
@@ -78,20 +79,19 @@ export interface GameStatus {
     | 'done'
     | 'canceled'
     | 'moving'
-  progress?: number | null
 }
 
 export interface InstallProgress {
+  bytes: string,
+  eta: string,
   percent: string
-  bytes: string
-  eta: string
 }
 export interface InstalledInfo {
   executable: string | null
-  version: string | null
+  install_path: string | null,
   install_size: string | null
-  install_path: string | null
-  is_dlc: boolean | null
+  is_dlc: boolean | null,
```

Comment:
Have noticed these trailing commas here and there.
It is weird to have them on interfaces though so I did set up my VSCode to remove trailing comma when formating and I think there is also a linting/prettier rule to remove them as well.

Reply:
I did as well, kind of weird.

Reply:
Fixed.